core/controlplane/config/templates/stack-template.json

```diff
@@ -203,12 +203,13 @@
                   "Resource": [ "*" ]
                 },
                 {{end}}
-                {{if .Experimental.ClusterAutoscalerSupport.Enabled}}
+                {{if .Addons.ClusterAutoscaler.Enabled}}
```

Comment:
Thanks! So it permits an action only when the targeted resource has the tag? Great - I will incorporate it.